tests/model_serving/model_runtime/vllm/utils.py

```
@@ -227,3 +234,90 @@ def validate_raw_tgis_inference_request(
+# Performance Helper Methods
+
+
+def get_vllm_version(namespace, pod_name):
+    cmd = f'oc exec -n {namespace} {pod_name} -- python -c "import vllm; print(vllm.__version__)"'
+    result = subprocess.check_output(cmd, shell=True, text=True)
+    return result.strip()
+
+
+def get_vllm_throughput_logs(namespace, pod_name):
+    cmd = f"oc logs -n {namespace} {pod_name} | grep 'Avg prompt throughput'"
+    result = subprocess.getoutput(cmd)
```

Ruff (0.15.15)

[error] Starting a process with a shell, possible injection detected

(S605)